services/emailer/devops/vaults.gcp.env

```diff
@@ -22,4 +22,6 @@ BUSINESS_REGISTRY_URL="op://web-url/$APP_ENV/business-registry-ui/BUSINESS_REGIS
 REPORT_API_URL="op://API/$APP_ENV/report-api/REPORT_API_URL"
 REPORT_API_VERSION="op://API/$APP_ENV/report-api/REPORT_API_VERSION"
 LEGAL_API_URL="op://API/$APP_ENV/legal-api/LEGAL_API_URL"
+LEGAL_API_VERSION="op://API/$APP_ENV/legal-api/LEGAL_API_VERSION"
+LD_SDK_KEY="op://launchdarkly/$APP_ENV/namex/LD_SDK_KEY"
```

Comment:
The LD SDK KEY is actually named differently:

![image](https://github.com/user-attachments/assets/c2fd7ac8-c806-4edc-b3ec-34d85eaa80b7)

Reply:
Changes look good. Resolving.